internal/integration/provision/upgrade.go

```diff
@@ -73,20 +73,18 @@ type upgradeSpec struct {
 }
 
 const (
-	previousRelease = "v1.0.6"
-	stableRelease   = "v1.1.0" // or soon-to-be-stable
+	previousRelease = "v1.1.2"
+	stableRelease   = "v1.2.0-beta.1" // or soon-to-be-stable
 	// The current version (the one being built on CI) is DefaultSettings.CurrentVersion.
 
-	previousK8sVersion = "1.23.6" // constants.DefaultKubernetesVersion in the previousRelease
-	stableK8sVersion   = "1.24.2" // constants.DefaultKubernetesVersion in the stableRelease
+	previousK8sVersion = "1.24.3"      // constants.DefaultKubernetesVersion in the previousRelease
+	stableK8sVersion   = "1.25.0-rc.1" // constants.DefaultKubernetesVersion in the stableRelease
```

Comment:
`-rc.1` or 1.25?

Reply:
stable is 1.2.0-beta.1, so whatever is there ;)

we'll test the upgrade from rc-1 to final 1.25.0 in the tests